contributing.rst

```diff
@@ -1,7 +1,49 @@
-The git stuff
+The Git stuff
 =============
 
-`Guide by github, scroll down a bit <https://guides.github.com/activities/contributing-to-open-source/>`_
+General commit rules
+--------------------
+
+1. All changes introduced by the commit (diff lines) must be related to the
```

Comment:
Change to commits. I prefer more commits over squashing. Squashing makes it hard to follow the PR discussions.